t/24-worker.t

```diff
@@ -39,6 +39,11 @@ use OpenQA::Worker::Jobs;
 use OpenQA::WebSockets::Server;
 use OpenQA::Schema::Result::Workers ();
 
+ok(
```

Comment:
@dasantiago My concern is that I'm seeing only a positiveunit  test scenario, but how about adding a test for when the worker and websocket have incompatible versions?.

Reply:
@foursixnine That test is only to check if the developer didn't forgot to change the version in two places.

The tests you're looking are here:
https://github.com/os-autoinst/openQA/pull/1552/files#diff-ddc22c6228b8445228e241f746647170R85